app/views/home/about.html.slim

```diff
@@ -198,3 +175,4 @@
         h5 Business | Alumni
       .ushover
         h4#class Class of 2016
+Contact GitHub 
```

Comment:
Also, for all these files make sure there's a newline at the end. That's what the circle and arrow symbol is pointing out -- there isn't one right now.